debugger/src/tpage.c

```diff
@@ -266,11 +267,12 @@ void tpage_pack_widgets(gboolean tabbed)
 	if (oldroot)
 	{
 		int i = 0;
-		while (widgets[i])
+		do
 		{
+			widget = widgets[i];
 			g_object_unref(*widgets[i]);
 			i++;
-		}
+		} while (widget);
```

Comment:
what's the point here?  You're not using the variable you set in the loop.

Reply:
Being confused by rewriting the code in many variants in too short time or so.
No idea what's happened to me there...:(